go/controller/autoscaler/autoscaler.go

```diff
@@ -52,8 +52,9 @@ func (j job) Fulfillment() float64 {
 type Autoscaler struct {
 	ticker  *time.Ticker
 	cluster Cluster
-	jobs    map[string]job
-	mutex   *sync.Mutex
+
+	mutex sync.Mutex
```

Comment:
It's a convention to define mutex on top of the vars that needs to be protected.